beetsplug/duplicates.py

```diff
@@ -253,13 +253,12 @@ def _order(self, objs, tiebreak=None):
         "completeness" (objects with more non-null fields come first)
         and Albums are ordered by their track count.
         """
-        if tiebreak:
-            kind = 'items' if all(isinstance(o, Item)
-                                  for o in objs) else 'albums'
+        kind = 'items' if all(isinstance(o, Item) for o in objs) else 'albums'
+
+        if tiebreak and kind in tiebreak.keys():
             key = lambda x: tuple(getattr(x, k) for k in tiebreak[kind])
         else:
-            kind = Item if all(isinstance(o, Item) for o in objs) else Album
-            if kind is Item:
+            if kind is 'items':
```

Comment:
I think the proper comparison operator on strings is `==`, not `is`.

Reply:
Yes, it is... oops.